apps/meteor/server/settings/e2e.ts

```diff
@@ -16,21 +16,19 @@ export const createE2ESettings = () =>
 			enableQuery: { _id: 'E2E_Enable', value: true },
 		});
 
-		await this.add('E2E_Enable_Encrypt_Files', true, {
+		await this.add('E2E_Enabled_Default_DirectRooms', false, {
 			type: 'boolean',
-			i18nLabel: 'E2E_Enable_Encrypt_Files',
-			i18nDescription: 'E2E_Enable_Encrypt_Files_Description',
 			public: true,
 			enableQuery: { _id: 'E2E_Enable', value: true },
 		});
 
-		await this.add('E2E_Enabled_Default_DirectRooms', false, {
+		await this.add('E2E_Enabled_Default_PrivateRooms', false, {
 			type: 'boolean',
 			public: true,
 			enableQuery: { _id: 'E2E_Enable', value: true },
 		});
 
-		await this.add('E2E_Enabled_Default_PrivateRooms', false, {
+		await this.add('E2E_Enable_Encrypt_Files', true, {
```

Comment:
⚠️ Here the settings' IDs are changed, therefore the olds records will remain at the database and the new ones will use default values. It's different from other situation we've discussed, where other fields would be updated without data loss.

Reply:
Only the order changed, not the ids

Reply:
<img src="https://media3.giphy.com/media/f1ohcPEHABwWY/giphy.gif"/>